iOS/Flows/Conversation/Emotions/EmotionContentCell.swift

```diff
@@ -14,61 +14,70 @@ struct EmotionContentModel: Hashable {
+
+        self.circleView.addSubview(self.definitionLabel)
+        self.definitionLabel.textAlignment = .center
         
-        self.contentView.addSubview(self.emotionLabel)
+        self.circleView.addSubview(self.emotionLabel)
     }
     
     func configure(with item: EmotionContentModel) {
         if let emotion = item.emotion {
-            
             let text = localized(emotion.definition).firstCapitalized
             let color = emotion.color
 
-            self.label.setText(text)
-            self.label.textColor = color
-            self.label.alpha = 1.0
-            self.contentView.layer.borderColor = color.cgColor
-            self.contentView.backgroundColor = color.withAlphaComponent(0.2)
+            self.definitionLabel.setText(text)
+            self.definitionLabel.textColor = color
+            self.definitionLabel.alpha = 1.0
+            self.circleView.layer.borderColor = color.cgColor
+            self.circleView.backgroundColor = color.withAlphaComponent(0.2)
             
             self.emotionLabel.setText(emotion.description)
             self.emotionLabel.textColor = color
             
         } else {
             self.emotionLabel.setText("")
-            self.label.setText("Select any emotion below to see what it means and add it to your message")
-            self.label.setTextColor(.white)
-            self.label.alpha = 0.2
-            self.contentView.set(backgroundColor: .clear)
-            self.contentView.layer.borderColor = ThemeColor.white.color.withAlphaComponent(0.2).cgColor
+            self.definitionLabel.setText("Select any emotion below to see what it means and add it to your message")
+            self.definitionLabel.setTextColor(.white)
+            self.definitionLabel.alpha = 0.2
+            self.circleView.set(backgroundColor: .clear)
+            self.circleView.layer.borderColor = ThemeColor.white.color.withAlphaComponent(0.2).cgColor
         }
         
         self.setNeedsLayout()
     }
     
     override func layoutSubviews() {
         super.layoutSubviews()
-        
-        self.label.setSize(withWidth: self.contentView.width * 0.8)
-        self.label.centerOnXAndY()
-        
-        self.emotionLabel.setSize(withWidth: self.contentView.width)
+
+        self.circleView.squaredSize = min(self.contentView.width, self.contentView.height)
+        self.circleView.centerOnXAndY()
+        self.circleView.makeRound()
+
+        self.definitionLabel.setSize(withWidth: self.circleView.width * 0.8)
+        self.definitionLabel.centerOnXAndY()
+
+        self.emotionLabel.setSize(withWidth: self.circleView.width)
+
+        self.definitionLabel.top += self.emotionLabel.height
+
         self.emotionLabel.centerOnX()
-        self.emotionLabel.match(.bottom, to: .top, of: self.label, offset: .negative(.standard))
+        self.emotionLabel.match(.bottom, to: .top, of: self.definitionLabel, offset: .negative(.standard))
```

Comment:
Some layout adjustments to the text. Now the title and definition are vertically centered as a unit.